Utility/Remove-ResourceGroups.ps1

```diff
@@ -0,0 +1,228 @@
+﻿
+<#PSScriptInfo
+
+.VERSION 1.0
+
+.GUID 
+
+.AUTHOR Azure-Automation-Team
+
+.COMPANYNAME Microsoft
+
+.COPYRIGHT 
+
+.TAGS AzureAutomation Utility
+
+.LICENSEURI 
+
+.PROJECTURI https://github.com/azureautomation/runbooks/blob/master/Utility/Remove-ResourceGroups.ps1
+
+.ICONURI 
+
+.EXTERNALMODULEDEPENDENCIES 
+
+.REQUIREDSCRIPTS 
+
+.EXTERNALSCRIPTDEPENDENCIES 
+
+.RELEASENOTES
+
+#>
+<# 
+.SYNOPSIS 
+  Connects to Azure and removes all resource groups which match the name filter
```

Comment:
How about "Connects to Azure and removes all resource groups which match the name filter.  
Warning: all resources inside the resource group will be deleted including child resources of that resource group."?

Don't want to scare people off - remember the default is that the preview is set to true...